### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Nelofarzabi**
The Requirements of this project were as follows: 

- To build a Desktop Version of the Portfolio similar to the one provided in the figma design.
- To use media queries in the CSS file to make the desktop version.
- To use Grid and Flexbox to layout elements on the page.

# Outcome

- Successfully built a desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,779 @@
...
+  }
+
+  .tech-name {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .title-desktop {
+    color: var(--h3-color);
+    font-size: 32px;
```
Comment (PinkMoon25) [inline style.css:719]: - [ ] In order to match the Figma design as much as possible let's also fix the font-size of each of the project cards title, currently it is 32px while on the figma it is 40px :+1:

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @Nelofarzabi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Media query :heavy_check_mark: 
Nice UI :heavy_check_mark: 
Good PR & Commits :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,779 @@
...
+    line-height: 28px;
+    color: #344563;
+    width: 97%;
+  }
+
+  .about-section {
+    display: grid;
+    gap: 25px;
+    grid-template-columns: 40% 1fr;
+    padding: 7.125rem 1.5rem;
```
[the remark this answers is not in the document]
Comment (Nelofarzabi) [reply style.css]: Hi @PinkMoon25 I have added the padding thank you for the comment.

File: style.css (hunk 1 above)
Comment (Nelofarzabi) [reply style.css]: Hi @PinkMoon25 I have increased the font size to 40px. thank you for the comment

File: README.md [hunk 3]
```diff
@@ -1 +1,121 @@
...
+
+I would like to thank from Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Nelofarzabi/Hello_Microverse/blob/dev/LICENSE) licensed.
```
[the remark this answers is not in the document]
Comment (Nelofarzabi) [reply README.md]: HI @PinkMoon25 I have fixed the broken link. thank you for your comment.

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @Nelofarzabi 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**